omod/src/main/java/org/openmrs/module/mirebalais/page/controller/patientRegistration/AfterRegistrationPageController.java

```diff
@@ -63,9 +62,9 @@ private String getRedirectUrlForLiberia(Config config, Patient patient) {
 
         String liberiaEmrId = pi.getIdentifier();
         if (liberiaEmrId != null && liberiaEmrId.startsWith(config.getPrimaryIdentifierPrefix())) {
-            redirectUrl += determineHtmlFormPath(config, "liveCheckin") + "&createVisit=true";
+            redirectUrl += PihCoreUtil.getFormResource("liveCheckin.xml") + "&createVisit=true";
         } else {
-            redirectUrl += determineHtmlFormPath(config, "checkin");
+            redirectUrl += PihCoreUtil.getFormResource("checkin.xml");
```

Comment:
Can we plan to add this to the pih-config, Mark? Like a `postRegistrationForm` property or something? Would love to see this branch abolished.

Reply:
Sure, seems reasonable to tackle at some point.